civihr_employee_portal/css/custom.css

```diff
@@ -269,6 +275,14 @@ span.appraisals-employee-legend {
   width: calc(100% - 30px);
 }
 
+#reportPivotTable .report-fields-selection table {
+  display: block;
+  width: 100%;
+  max-height: 50vh;
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
List properties in alphabetical order.

Reply:
The properties are not alphabetically arranged!  Not sure why you move `overflow-x: hidden;` to last.
It should be like following no?
```css
#reportPivotTable .report-fields-selection table {
    display: block;
    max-height: 50vh;
    width: 100%;
    overflow-x: hidden;
    overflow-y: auto;
}
```

Reply:
I think it should be like following:
```css
#reportPivotTable .report-fields-selection table {
  display: block; 
  max-height: 50vh;
  overflow-x: hidden;
  overflow-y: auto;
  width: 100%;
}
```
Not sure why you moved `overflow-x: hidden;` to the last.